packages/cashier/src/Containers/crypto-deposit.jsx

```diff
@@ -71,11 +71,11 @@ const CryptoDeposit = ({
                     {header_note}
                 </Text>
 
-                {api_error ? (
+                {!api_error ? (
```

Comment:
Aren't we showing the error content when `api_error` is `true`/`truthy`?

Reply:
My bad, used that flag for debugging, i'm sorry